docs/src/4.10.Text.mdx

```diff
@@ -23,15 +49,18 @@ type Text = {
     y: number,
     z: number
   },
-  color: {
-    r: number, // between 0 and 1
-    g: number, // between 0 and 1
-    b: number, // between 0 and 1
-    a: number, // between 0 and 1
-  },
+  color: Color,
```

Comment:
So this `color` field is always required, even when you just want to pass in a 2 element `colors` array?

Reply:
My instinct is to just get rid of this first `color` prop, and just conditionally check if the `colors` array has 1 or 2 elements?

Reply:
good instinct on getting rid of `color`, but we should support both single color as the `color` prop as well as multiple.  ROS markers support both [color and colors](http://docs.ros.org/api/visualization_msgs/html/msg/Marker.html).  Also for backwards compatibility reasons we should maintain support for `color` - though you're right it should be marked as an optional, not required, property in flow.

Reply:
Good feedbacks. Marked the color as optional